### DISCUSSION TITLE
How to start Martin connected to mbtiles and PostGIS?

### DISCUSSION THREAD

**Discussion by gagecarto** [maplibre/martin#1551] [Q&A]
I am currently running a Martin instance connected to several mbtiles like below

```
./martin someTiles.mbtiles someMoreTiles.mbtiles
```

Could I start Martin connected to both local mbtiles and a PostGIS connection string?

**Comment by nyurik** [comment]
sure - just add a connection string at the end:

`./martin someTiles.mbtiles postgres://pghost/dbname`  or use `DATABASE_URL` env var.  I would also recommend using `--save-config myconfig.yaml` -- this way you can auto-generate a config file that you can later customize and use
